crates/re_ui/src/command.rs

```diff
@@ -143,6 +145,11 @@ impl UICommand {
                 "Screenshot",
                 "Copy screenshot of the whole app to clipboard",
             ),
+            #[cfg(not(target_arch = "wasm32"))]
+            UICommand::DumpDatastore => (
+                "Dump datastore",
+                "Dumps the current state of the datastore to the console",
```

Comment:
```suggestion
                "Print datastore",
                "Prints the entire data store to the console. WARNING: this may be A LOT of text.",
```